media/html/visjsDebuggerPanel.html

```diff
@@ -324,6 +361,49 @@
       function openAllClusters() {
         openClusterStack(Object.keys(network.body.nodes).filter((nodeId) => nodeId.startsWith('cluster_')));
```

Comment:
```suggestion
        openClusterStack(network ? Object.keys(network.body.nodes).filter((nodeId) => nodeId.startsWith('cluster_')) : []);
```
The button is already interactable before the network is created.